src/index.js

```diff
@@ -240,6 +246,9 @@ ConnectedWidget.defaultProps = {
   showHeaderAvatar: true,
   sessionId: null,
   startFullScreen: false,
+  suggestionsUrl: 'https://api-staging.bothub.it/v2/repository/examples/search_repositories_examples/',
```

Comment:
Can you change the default value for URL for https://api.bothub.it instead of https://api-staging.bothub.it